travel-event-host-app/src/components/categories/categories.tsx

```diff
@@ -0,0 +1,15 @@
+import styles from './styles.module.css';
+import categoriesArr from '@/lib/categoryArray';
+
+export default function Categories() {
+  return (
+    <section className={styles.section}>
+      <h2>Categories</h2>
+      <ul className={styles.categoriesUl}>
+        {Object.values(categoriesArr).map((category) => (
```

Comment:
You can use the `CategoryDict` here with the `Object.values`
```suggestion
        {Object.values(CategoryDict).map((category) => (
```